private/model/api/passes.go

```diff
@@ -244,6 +244,46 @@ func (a *API) renameExportable() {
 	}
 }
 
+// renameCollidingFields will rename any fields that uses an SDK or Golang
+// specific name.
+func (a *API) renameCollidingFields() {
+	for _, v := range a.Shapes {
+		namesWithSet := map[string]struct{}{}
+		for k, field := range v.MemberRefs {
+			if strings.HasPrefix(k, "Set") {
+				namesWithSet[k] = struct{}{}
+			}
+
+			if collides(k) {
+				delete(v.MemberRefs, k)
+				v.MemberRefs[k+"_"] = field
+			}
+		}
+
+		// checks if any field names collide with setters.
+		for name := range namesWithSet {
+			setterName := fmt.Sprintf("Set%s", name)
+			if field, ok := v.MemberRefs[setterName]; ok {
```

Comment:
Instead of a `fmt.Sprintf` in this case i think it would be better just to do `v.MemberRefs["Set"+name]`.  Not really gaining anything with the fmt. This is the pattern used for the actual renaming logic.